docusaurus/docs/e2e-test-a-plugin/getting-started.md

```diff
@@ -0,0 +1,157 @@
+
+## Set up `plugin-e2e`
+
+### Step 1: Installing @grafana/plugin-e2e
+
+Now open the terminal and run the following command in your plugin's project directory:
+
+<CodeSnippets
+snippets={[
+{ component: ScaffoldPluginE2InstallNPM, label: 'npm' },
+{ component: ScaffoldPluginE2EInstallYarn, label: 'yarn' },
+{ component: ScaffoldPluginE2EInstallPNPM, label: 'pnpm' }
+]}
+groupId="package-manager"
+queryString="current-package-manager"
+/>
+
+### Step 2: Configure Playwright
+
+Open the Playwright config file that was generated when Playwright was installed.
+
+1. Uncomment `baseUrl` and change it to `'http://localhost:3000'`.
+
+```ts title="playwright.config.ts"
+  baseURL: 'http://localhost:3000',
+```
+
+2. Playwright uses [projects](https://playwright.dev/docs/test-projects) to logically group tests that have the same configuration. We're going to add two projects. The first project called `auth` is a setup project will login to Grafana and store the authenticated state on disk. The second project `run-tests` runs all the tests in a browser of choice. By adding a dependency to the `auth` project we ensure login only happens once, and all tests in the `run-tests` project will start already authenticated.
+
+   Your Playwright config should have the following project configuration:
+
+<CodeSnippets
+snippets={[
+{ component: ScaffoldPluginE2EProjectsConfigNPM, label: 'npm' },
+{ component: ScaffoldPluginE2EProjectsConfigYarn, label: 'yarn' },
+{ component: ScaffoldPluginE2EProjectsConfigPNPM, label: 'pnpm' }
+]}
+groupId="package-manager"
+queryString="current-package-manager"
+/>
+
+```ts title="playwright.config.ts"
+projects: [
+  {
+    name: 'auth',
+    testDir: 'node_modules/@grafana/plugin-e2e/dist/auth',
+    testMatch: [/.*\.js/],
+  }
+```
+
+### Step 3: Start Grafana
+
+Next, start up the latest version of Grafana on your local machine.
+
+<CodeSnippets
+snippets={[
+{ component: ScaffoldPluginE2EStartGrafanaNPM, label: 'npm' },
+{ component: ScaffoldPluginE2EStartGrafanaYarn, label: 'yarn' },
+{ component: ScaffoldPluginE2EStartGrafanaPNPM, label: 'pnpm' }
+]}
+groupId="package-manager"
+queryString="current-package-manager"
+/>
+
+If you want to start a specific version of Grafana, you can do that by specifying the `GRAFANA_VERSION` environment variable.
+
+```bash
+GRAFANA_VERSION=10.1.6 npm run server
+```
+
+### Step 4: Write tests
+
+In this example, we're using the panel edit page to test a data source plugin. When the provided query is valid, the response status code is expected to be in the range 200-299.
+
+```ts title="queryEditor.spec.ts"
+import { test, expect } from '@grafana/plugin-e2e';
+
+test('data query should return values 1 and 3', async ({ panelEditPage, readProvisionedDataSource }) => {
+  const ds = await readProvisionedDataSource({ fileName: 'datasources.yaml' });
```

Comment:
The [pr](https://github.com/grafana/plugin-tools/pull/740) that adds this fixture is not merged yet